src/frontend/src/components/WebformTable/components/EditTable/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Table as MuiTable, TableBody, TableHead } from "@material-ui/core";
+import React from "react";
+import { SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS } from "../../common/constants";
+import {
+  IsPrivateTableCell,
```

Comment:
I don't see the style sheet in your branch. But this probably shouldn't start with `Is`, since it's a styled component. Normally, would only reserve that for booleans.

Reply:
This is a different case since this is actually referring to the IsPrivate metadata field, i would vote that we keep it since it makes sense with the naming of that field if that's ok with you.

Reply:
the stylesheet is included here: https://github.com/chanzuckerberg/czgenepi/blob/87bf3824bae0c77abae447d35aa0ec9579d10d34/src/frontend/src/components/WebformTable/style.ts#L47:14

Reply:
i also didn't write either of those styled components, they already exist for styling the upload metadata table, i just changed where they were imported